server/utils/dateHelpers.ts

```diff
@@ -90,7 +90,7 @@ export const isRealDate = (date: string): boolean => {
 export const formatDate = (isoDate: string, style: 'short' | 'full' | 'long' | 'medium' = 'long'): string => {
   if (!isoDate) return ''
 
-  return new Date(isoDate).toLocaleDateString('en-gb', { dateStyle: style })
+  return new Date(isoDate).toLocaleDateString('en-gb', { dateStyle: style }).replace(',', '')
 }
```

Comment:
Assume this is ok... doesn't seem to be being used anwhere in 'full' form

Reply:
I thought when David updated the version of node here: https://github.com/ministryofjustice/hmpps-digital-prison-services/commit/cff4d4240f512fccf1502620c78c76a2dc2103ff#diff-a408d0865fe53136a5838fbb143751bdbdbb4ece44c9ab76920a860740a67ddb

That this stopped adding the `,` in?

I wonder if you need this if you do a `nvm install --latest-npm` to make sure you have 20.12?

Reply:
Correct, I'd made the school boy error of not pulling the latest version before working on it